doc/7-systems/snes.md

```diff
@@ -65,6 +65,18 @@ Furnace also allows the SNES to use wavetables (and the wavetable synthesizer) i
 
 this chip uses the [SNES](../4-instrument/snes.md) instrument editor.
 
+## channel status
+
+when enabled, channel status will show an additional icon representing the gain mode and envelope status of the currently playing note:
+- ![direct gain, envelope off](status-SNES-gain-direct.png) direct gain, envelope off
+- ![linear gain decrease](status-SNES-gain-dec-lin.png) linear gain decrease
+- ![logarithmic gain decrease](status-SNES-gain-dec-log.png) logarithmic gain decrease
```

Comment:
exponential (??)
Sadly I don't remember whether I called it log or exp in the ins editor, so feel free to ignore and mark as resolved if I called it log.